pt-BR/mail/chrome/messenger/phishing-afterload-warning-message.dtd

```diff
@@ -1,28 +1,28 @@
+<!ENTITY safeb.palm.accept.label "Me tire daqui!">
+<!ENTITY safeb.palm.accept.statustext "Navegar para minha página inicial">
+<!ENTITY safeb.palm.decline.label "Ignorar este aviso">
+<!ENTITY safeb.palm.decline.statustext "Fechar aviso" >
+<!ENTITY safeb.palm.notforgery.label2 "Não é uma falsificação…">
+<!ENTITY safeb.palm.report.label "Por que este site foi bloqueado?">
 
-<!ENTITY safeb.urlbaricon.tooltip "This page might be dangerous; click for details.">
+<!ENTITY safeb.urlbaricon.tooltip "Esta página pode ser perigosa. Clique para mais detalhes.">
 
-<!ENTITY safeb.blocked.malware.title "Reported Attack Site!">
+<!ENTITY safeb.blocked.malware.title "Site de ataque reportado!">
 <!-- Localization note (safeb.blocked.malware.shortDesc) - Please don't translate the contents of the <span id="malware_sitename"/> tag.  It will be replaced at runtime with a domain name (e.g. www.badsite.com) -->
-<!ENTITY safeb.blocked.malware.shortDesc "This web site at <span id='malware_sitename'/> has been reported as an attack site and has been blocked based on your security preferences.">
-<!ENTITY safeb.blocked.malware.longDesc "<p>Attack sites try to install programs that steal private information, use your computer to attack others, or damage your system.</p><p>Some attack sites intentionally distribute harmful software, but many are compromised without the knowledge or permission of their owners.</p>">
+<!ENTITY safeb.blocked.malware.shortDesc "O site em <span id='malware_sitename'/> foi reportado como um site que realiza ataques e foi bloqueado baseado nas suas preferências de segurança.">
+<!ENTITY safeb.blocked.malware.longDesc "<p>Sites que realizam ataques tentam instalar programas que roubam informações privadas, utlizam seu computador para atacar outros usuários ou danificam o seu sistema.</p><p>Alguns sites distribuem de forma intencional software malicioso, mas muitos estão comprometidos sem conhecimento ou permissão dos donos.</p>">
 
-<!ENTITY safeb.blocked.phishing.title "Reported Web Forgery!">
+<!ENTITY safeb.blocked.phishing.title "Falsificação na Internet reportada!!">
 <!-- Localization note (safeb.blocked.phishing.shortDesc) - Please don't translate the contents of the <span id="phishing_sitename"/> tag. It will be replaced at runtime with a domain name (e.g. www.badsite.com) -->
-<!ENTITY safeb.blocked.phishing.shortDesc "This web site at <span id='phishing_sitename'/> has been reported as a web forgery and has been blocked based on your security preferences.">
-<!ENTITY safeb.blocked.phishing.longDesc "<p>Web forgeries are designed to trick you into revealing personal or financial information by imitating sources you may trust.</p><p>Entering any information on this web page may result in identity theft or other fraud.</p>">
+<!ENTITY safeb.blocked.phishing.shortDesc "O site em <span id='phishing_sitename'/> foi reportado como uma falsificação na Internet e foi bloqueado baseado nas suas preferências de segurança.">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY safeb.blocked.phishing.shortDesc "O site em <span id='phishing_sitename'/> foi reportado como uma falsificação na Internet e foi bloqueado com base nas suas preferências de segurança.">